chart/compass/charts/gateway/templates/virtualservice.yaml

```diff
@@ -90,6 +90,7 @@ spec:
       corsPolicy:
         allowOrigins:
           - regex: "(http|https)://(compass-dev|compass|compass-mf).{{ .Values.global.ingress.domainName }}(:[0-9]{4})?"
+          - regex: "(http|https)://localhost(:[0-9]{4})?"
```

Comment:
I think ports go all the way till 65535. So maybe `[0-9]{5}` would be preferable?